apps/web/src/env.mjs

```diff
@@ -26,34 +44,11 @@ export const env = createEnv({
     DATABASE_URL: z.string().url(),
     FEEDBACK_WEBHOOK_URL: z.string().optional(),
     NODE_ENV: z.enum(["development", "test", "production"]),
-    METRICS_ENABLED: booleanSchema.default("false"),
-    TRACES_ENABLED: booleanSchema.default("false"),
+    METRICS_ENABLED: z.boolean().default(false),
+    TRACES_ENABLED: z.boolean().default(false),
   },
-  /**
-   * Specify your client-side environment variables schema here.
-   * For them to be exposed to the client, prefix them with `NEXT_PUBLIC_`.
-   */
   client: {
```

Comment:
We should avoid redeclaring all the env vars in the `client` field again, which is what we're trying to avoid by exposing them from the server.

Upon further inspection, I realise that there're some of them we should keep defined at build time. Those are:

- `NEXT_PUBLIC_BLOBSCAN_RELEASE`: This one we actually set it when building the docker image.
-  `NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA`: This is a default variable that Vercel uses so we can't remove the prefix